lib/charms/mongodb/v1/shards_interface.py

```diff
@@ -145,6 +145,13 @@ def _on_relation_joined(self, event):
 
         self.database_provides.update_relation_data(event.relation.id, relation_data)
 
+    def __handle_relation_not_feasible(self, event: EventBase):
```

Comment:
**nit:** though the `__` is commonly used for name mangling of "private" methods, it is not really necessary in the context of charms / charm libs, where inheritance is performed rather modestly, and where most of the client code is actual charm components aware of what (not) to use

Reply:
Sure, I'll rename it then :)